Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.iOS/AppDelegate.cs

```diff
@@ -1,15 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Contoso.Forms.Demo.iOS;
 using Foundation;
 using Microsoft.AppCenter.Distribute;
 using UIKit;
+using Xamarin.Forms;
 
+[assembly: Dependency(typeof(AppDelegate))]
 namespace Contoso.Forms.Demo.iOS
 {
     [Register("AppDelegate")]
-    public class AppDelegate : Xamarin.Forms.Platform.iOS.FormsApplicationDelegate
+    public class AppDelegate : Xamarin.Forms.Platform.iOS.FormsApplicationDelegate, IClearCrashClick
     {
+        public const string kMSUserConfirmationKey = "MSUserConfirmation";
```

Comment:
why public, also on c# constants don't have kMS prefix.

Please rename to `CrashesUserConfirmationStorageKey`